CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * There's now a virtual root operation, similar to the [virtual root
   commit](docs/glossary.md#root-commit). It appears at the end of `jj op log`.
 
+* `jj config list` gained a `--include-overridden` option to allow
+  printing overridden config values
```

Comment:
nit: missing period (we seem to quite consistently have period at end of sentence here)